ts/dx.all.d.ts

```diff
@@ -9475,6 +9475,11 @@ declare module DevExpress.ui {
          * [descr:dxOverlayOptions.dragEnabled]
          */
         dragEnabled?: boolean;
+        /**
```

Comment:
`dx.all.d.ts` is outdated now. U should run `npm run update-ts`

Reply:
done